_inc/client/at-a-glance/search.jsx

```diff
@@ -55,6 +56,20 @@ class DashSearch extends Component {
 		isDevMode: false,
 	};
 
+	constructor() {
+		super( ...arguments );
+
+		this.trackSearchLink = this.trackSearchLink.bind( this );
```

Comment:
Same here. We don't really need this binding as `trackSearchLink` makes no reference to `this` in its code.